### PR TITLE
Add pdf

### PR DESCRIPTION
**Pull request by TuomasBorman**
Hi,

this adds pdf version.

Issue: https://github.com/microbiome/OMA/issues/140

-Tuomas

### CODE REVIEW

**Review by antagomir — APPROVED** [review]
!

**Comment by antagomir** [comment]
just merge when checks pass

File: 23_multi-assay_analyses.Rmd [hunk 1]
```diff
@@ -182,7 +182,7 @@ if(!require(reticulate)){
 }
 
 reticulate::install_miniconda(force = TRUE)
-reticulate::use_miniconda()
+reticulate::use_miniconda(condaenv = "env1", required = FALSE)
 reticulate::py_install(packages = c("mofapy2"), pip = TRUE)
 ```
```
Comment (TuomasBorman) [inline 23_multi-assay_analyses.Rmd:187]: Hopefully fixes 
image

File: 30_differential_abundance.Rmd [hunk 2]
```diff
@@ -543,7 +543,7 @@ Automatically initializing the priors and visualizing their distributions:
 ```{r}
 priors <- pibble(NULL, X, upsilon, Theta, Gamma, Xi)
 names_covariates(priors) <- rownames(X)
-fido::plot(priors, pars="Lambda") + ggplot2::xlim(c(-5, 5))
+plot(priors, pars="Lambda") + ggplot2::xlim(c(-5, 5))
 ```
```
Comment (TuomasBorman) [inline 30_differential_abundance.Rmd:548]: Fixes 
image

File: index.Rmd [hunk 3]
```diff
@@ -6,6 +6,7 @@ bibliography: [book.bib]
 biblio-style: apalike
 link-citations: yes
 github-repo: microbiome/OMA
+always_allow_html: true
 ---
```
Comment (TuomasBorman) [inline index.Rmd:10]: Fixes

image

Comment (TuomasBorman) [reply index.Rmd]: Removed since it did not work

File: _bookdown.yml [hunk 4]
```diff
@@ -25,14 +25,12 @@ rmd_files: ["index.Rmd",
...
             "21_microbiome_community.Rmd",
-            "24_biclustering.Rmd",	    
+            "24_biclustering.Rmd",
 
             "30_differential_abundance.Rmd",
 
             "23_multi-assay_analyses.Rmd",
 
-            "24_biclustering.Rmd",
-
```
Comment (TuomasBorman) [inline _bookdown.yml:35]: Additional bicluster chapter removed

File: 24_biclustering.Rmd [hunk 5]
```diff
@@ -162,6 +162,9 @@ if(!require(biclust)){
     library(biclust)
 }
 
+# Set seed for reproducibility
+set.seed(3973)
+
 # Find biclusters
 bc <- biclust(corr, method=BCPlaid(), fit.model = y ~ m,
```
Comment (TuomasBorman) [inline 24_biclustering.Rmd:169]: So that we get same clusters every time

File: _output.yml [hunk 6]
```diff
@@ -21,3 +21,4 @@ bookdown::pdf_book:
   latex_engine: xelatex
   citation_package: natbib
   keep_tex: yes
+  always_allow_html: true
```
Comment (TuomasBorman) [inline _output.yml:24]: This should allow us to create pdf files even though there are html figures

Comment (TuomasBorman) [reply _output.yml]: Did not work

File: book.bib [hunk 7]
```diff
@@ -330,8 +330,7 @@ @Book{McElreath2020
   title =	 {Statistical Rethinking},
   publisher =	 {Chapman and Hall/CRC},
   year =	 2020,
-  note =	 {with implementation by Solomon Kurz:
-                  https://bookdown.org/ajkurz/Statistical_Rethinking_recoded/}
+  note =	 {with implementation by Solomon Kurz: https://bookdown.org/ajkurz/Statistical\_Rethinking\_recoded/}
 }
```
Comment (TuomasBorman) [inline book.bib:334]: Apparently, underscore causes problem so I changes the to "/_"

image

File: .github/workflows/build-book-release.yml [hunk 8]
```diff
@@ -179,6 +179,14 @@ jobs:
...
           Rscript -e 'bookdown::render_book("index.Rmd", "bookdown::gitbook")'
 
+      - name: Bookdown pdf
+        if: runner.os == 'Linux'
+        run: |
+          Rscript -e 'tinytex::install_tinytex()'
+          Rscript -e 'install.packages("webshot")'
+          Rscript -e 'webshot::install_phantomjs()'
+          Rscript -e 'bookdown::render_book("index.Rmd", "bookdown::pdf_book")'
+
```
Comment (TuomasBorman) [inline .github/workflows/build-book-release.yml:189]: New packages allow us to generate pdf even though there are html elements

File: index.Rmd [hunk 9]
```diff
@@ -6,6 +6,7 @@ bibliography: [book.bib]
 biblio-style: apalike
 link-citations: yes
 github-repo: microbiome/OMA
+always_allow_html: yes
 ---
```
Comment (TuomasBorman) [inline index.Rmd:10]: This is added so that we can create pdf when there are html elements

**Pull request merged**